### ISSUE TITLE
Some "MedicalTestPanel" sub-type missing from csv file

### ISSUE THREAD

**Issue by khalyomede** [schemaorg/schemaorg#1882]
The following types are missing from csv file `http://schema.org/version/latest/all-layers-types.csv`:

- `http://schema.org/OccupationalTherapy`
- `http://schema.org/RespiratoryTherapy`

This property is a sub-type of `http://schema.org/MedicalTestPanel`.

**Comment by RichardWallis** [comment]
I've had a look at the csv and indeed there is something odd.

 ```OccupationalTherapy``` and ```RespiratoryTherapy``` are subtypes of ```MedicalTherapy```, yet when inspecting their reference pages [[1](http://health-lifesci.schema.org/OccupationalTherapy)], [[2](http://health-lifesci.schema.org/RespiratoryTherapy)] they appear to be enumeration values for ```MedicalSpecialty```.

Things seem to be reported similarly in the equivalent Turtle file:
```
schema:OccupationalTherapy a schema:MedicalSpecialty ;
    rdfs:label "OccupationalTherapy" ;
    schema:isPartOf <http://health-lifesci.schema.org> ;
    rdfs:comment "A treatment of people with physical, emotional, or social problems, using purposeful activity to help them overcome or learn to deal with their problems." ;
    rdfs:subClassOf schema:MedicalTherapy .
```
 "```schema:OccupationalTherapy a schema:MedicalSpecialty```" yet "```rdfs:subClassOf schema:MedicalTherapy```" - doesn't seem to make sense, it needs looking into deeper to see if it is a term definition problem or an issue in the way that the processes that output these files are interpreting enumeration values.

I will label this as a bug.

**Comment by twamarc** [comment]
I agree @RichardWallis , this seems to be unfixed bug since the migration.
``` rdfs:subClassOf schema:MedicalTherapy```  is correct.
```schema:OccupationalTherapy a schema:MedicalSpecialty``` should be droped 
Here we confused the wording with ```OccupationalMedicine``` [not in schema.org] which is a ```MedicalSpecialty```.

I suggest to fix this asap -just like the ```PhysicalTherapy``` (they are both children of ```MedicalTherapy```) 

Suggested fix:
Current .rdfa file part for this:
```
<div typeof="http://schema.org/MedicalSpecialty" resource="http://schema.org/OccupationalTherapy">
  <span class="h" property="rdfs:label">OccupationalTherapy</span>
  <span property="rdfs:comment">A treatment of people with physical, emotional, or social problems, using purposeful activity to help them overcome or learn to deal with their problems.</span>
  <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/MedicalTherapy">MedicalTherapy</a></span>
<link property="http://schema.org/isPartOf" href="http://health-lifesci.schema.org" />
</div>
```

New .rdfa file part to replace the existing:
```
*<div typeof="rdfs:Class" resource="http://schema.org/OccupationalTherapy">*
  <span class="h" property="rdfs:label">OccupationalTherapy</span>
  <span property="rdfs:comment">A treatment of people with physical, emotional, or social problems, using purposeful activity to help them overcome or learn to deal with their problems.</span>
  <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/MedicalTherapy">MedicalTherapy</a></span>
<link property="http://schema.org/isPartOf" href="http://health-lifesci.schema.org" />
</div>
```
If you agree I can prepare a PR.

**Comment by danbri** [comment]
@RichardWallis can you generalize from this bug to a new unit test? It seems like the kind of mistake we ought to be programmatically checking for by now.

**Comment by RichardWallis** [comment]
Released in V3.4
